sim/paladin/paladin.go

```diff
@@ -232,7 +234,11 @@ func (paladin *Paladin) registerStopAttackMacros() {
 func (paladin *Paladin) ResetCurrentPaladinAura() {
 	paladin.currentPaladinAura = nil
 	if paladin.primaryPaladinAura == proto.PaladinAura_SanctityAura {
-		paladin.currentPaladinAura = core.SanctityAuraAura(paladin.GetCharacter())
+		if paladin.sanctityAura == nil {
+			paladin.sanctityAura = core.SanctityAuraAura(paladin.GetCharacter())
+		}
```

Comment:
We should probably just set `paladin.sanctityAura` in the initializer instead of here so that we don't have to check if it's `nil` like this